content/guides/core-concepts/introduction-to-cypress.md

```diff
@@ -1168,14 +1164,15 @@ Without a single explicit assertion, there are dozens of ways this test can
 fail! Here's a few:
 
 - The initial [`cy.visit()`](/api/commands/visit) could respond with something
-  other than success.
+  other than success. Or, the mounted component may be incorrectly imported.
```

Comment:
Can you make this into two separate bullet points?

Reply:
👍